### PR TITLE
Delete index dropdown

### LINKED ISSUE #735: Missing "delete index" action when browsing an index
## Expected Behavior
Be able to delete an index when browsing it

## Current Behavior
Missing "delete index" action when browsing an index

## Context (Environment)
Admin Console Version: 4

### PR DESCRIPTION
**Pull request by berthieresteban**
fix #735 

add actions dropdown on index view (collection list) to allow delete index 
acdeleteindex

### CODE REVIEW

File: src/components/Data/Indexes/DropdownActions.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<template>
+  <span>
+    <b-dropdown
+      v-if="
+        $store.direct.getters.kuzzle.currentEnvironment.backendMajorVersion !==
```
Comment (xbill82) [inline src/components/Data/Indexes/DropdownActions.vue:5]: please map this to a computed :)

File: src/components/Data/Indexes/DropdownActions.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        <b-dropdown-item
+          data-cy="IndexDropdown-delete"
+          :disabled="!canDeleteIndex(indexName)"
+          :title="
+            !canDeleteIndex(indexName)
+              ? 'Your rights do not allow you to delete this index'
+              : ''
+          "
+          v-if="
+            $store.direct.getters.kuzzle.currentEnvironment
```
Comment (xbill82) [inline src/components/Data/Indexes/DropdownActions.vue:28]: same same

**Review by xbill82 — CHANGES_REQUESTED** [review]
Good job! Just a few nitpicking!

File: src/components/Data/Indexes/DeleteIndexModal.vue [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
-    size="lg"
-    :id="modalId"
-    :ref="modalId"
-    @hide="resetForm"
-  >
+  <b-modal class="DeleteIndexModal" size="lg" :id="modalId" @hide="resetForm">
     <template v-slot:modal-title>
       Index
-      <strong>{{ truncateName(index.name) }}</strong> deletion
+      <strong>{{ truncateName(index ? index.name : '') }}</strong> deletion
```
[the remark this answers is not in the document]
Comment (berthieresteban) [reply src/components/Data/Indexes/DeleteIndexModal.vue:5]: `truncateName` is generic and is used in a lot of views for various names.
In `truncateName` the `name === null` state is already handled
Here I just test that `index` is empty so I'm sure to be able to read `index.name` which is different that if `index.name === null`

Not sure if I'm clear...

**Review by xbill82 — APPROVED** [review]

**Review by alexandrebouthinon — APPROVED** [review]

**Pull request merged**